### PR TITLE
CSV skills import

### LINKED ISSUE #320: Build an import pipeline for adding skills from CSV
We need a way to parse a CSV of skills and upload them. This ideally should be idempotent and can be used as a way for developers to seed the app, as well as seed our own production database.

### PR DESCRIPTION
**Pull request by schneidmaster**
Closes #320

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -32,4 +32,8 @@ vendor/bundle/*
 
 dump.rdb
 
-coverage
\ No newline at end of file
+coverage
+
+/orgnizations
+/projects
+/users
```
Comment (joshsmith) [inline .gitignore:39]: looks like it should be `/organizations`

Comment (schneidmaster) [inline .gitignore:39]: it should be, but it's misspelled in the model too, so 😛

Comment (joshsmith) [inline .gitignore:39]: Well. 😢 

Can we change in both then? I'll update in S3.

Comment (schneidmaster) [inline .gitignore:39]: Yeah will do

**Comment by joshsmith** [comment]
I renamed organizations folder in all S3 so we're good there.

File: app/policies/import_policy.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  def create?
+    # Cannot create if there's no user
+    return false unless user.present?
+
+    # Cannot create if they're not an admin.
+    return false unless user.admin?
+
+    # Can create import.
+    true
+  end
```
Comment (begedin) [inline app/policies/import_policy.rb:18]: I would just go with `return true if user.admin?` here. Sure, it does't follow our usual convention, but it's much more readable, I think.

**Comment by begedin** [comment]
@JoshSmith @schneidmaster 

I added 2 commits here

The first one is relatively simple. It adds a slug to the skill model, so we can, in the future, implement a skill page. Unlike users and organizations, the skills slug won't be accessed from the common slugged route (I assume) and will instead be on "skills/:slug" or something like that. Due to that, the behavior is not exactly the same.

The second commit adds handling of import failures. For each row that fails, it's status is set to 3 (`:failed`) and an `ImportSkillFailure` record is created, which has 4 columns:
- `skill` - the skill that was failed to update. it we failed at creating a new skill, this will be null
- `import` - the import record that failed
- `issues` - `skill.errors.full_messages`, a text array column
- `data`\- the csv row that failed to import. JSON column

All of these work just fine with our serializer.

I also added a very simple endpoint to fetch a list of failures, which can be enhanced as we need it.

**Comment by schneidmaster** [comment]
LGTM 👍

**Pull request merged**